corehq/apps/analytics/tasks.py

```diff
@@ -331,6 +334,10 @@ def track_periodic_data():
                 {
                     'property': 'project_spaces_created_by_user',
                     'value': project_spaces_created,
+                },
+                {
+                    'property': 'over_300_form_submissions',
+                    'value': forms_300
```

Comment:
Not blocking, but I think you can make this a little more concise by removing the the calculation of `forms_300` in the loop and just replacing this line with
```python
'value': max_forms > 300
```
(I'll merge before today's deploy regardless)